modules/apps/portal-search/portal-search-ranking-web/src/main/resources/META-INF/resources/js/components/ResultsRankingForm.es.js

```diff
@@ -477,10 +482,22 @@ class ResultsRankingForm extends Component {
 					state => (
 						{
 							dataLoading: false,
+
 							dataMap: {
-								...state.dataMap,
+
+								// Set the addedResult to false if it shows up as one
+								// of the results that pops up in search.
```

Comment:
The wording of this comment is a bit strange since we're saying "if" something shows up when in reality we're setting all duplicate items to false? 

Can we update the comments to be more descriptive? I think it's important to explain **why** we're setting these duplicate items to false since it's not clear how fetching items would need to set `addedResult` to false.